docs/en/File-System-API.md

```diff
@@ -82,30 +82,32 @@ Below is a table of the expected behaviors of `WriteType`
 
 ### Location policy
 
-Alluxio provides location policy to choose which workers to store the blocks of a file. User can set
-the policy in `CreateFileOptions` for writing files and `OpenFileOptions` for reading files into
-Alluxio. Alluxio supports custom location policy, and the built-in polices include:
+Alluxio provides location policy to choose which workers to store the blocks of a file. 
 
-* **LocalFirstPolicy**
+Using Alluxio's Java API, user can set the policy in `CreateFileOptions` for writing files and `OpenFileOptions` for reading files into
+Alluxio.
+
+Or to use the built-in policies, user can also simply override the default policy class in the [configuration file](Configuration-Settings.html) at property `alluxio.user.file.write.location.policy.class`. The built-in polices include:
```

Comment:
remove: "Or to use the built-in policies, "
user can also -> Users can
polices  -> policies